examples/PyOpenCL/test_build_dir

```diff
@@ -3,7 +3,12 @@
 # Runs the PyOpenCL tests against a pocl build dir located in the
 # current working directory. Assumes an ICD build in PWD.
 
-pyopencl_root=`dirname $0`
+pyopencl_root=$1
+if [ -z "$pyopencl_root" ]; then
+  pyopencl_root=`dirname $0`
+fi
+
+if [ -z "$OCL_ICD_VENDORS" ]; then
```

Comment:
This is dangerous. The idea of the script was to forcibly test the current build dir. Now, if the tester has set OCL_ICD_VENDORS (accidentally), it tests that instead?